docs/azure/index-old.yml

```diff
@@ -0,0 +1,201 @@
+brand: dotnet
+
+metadata:
+  title: Azure for .NET developers
+  description: Samples, tutorials, and education for .NET on Azure
+  ms.topic: hub-page
+  ms.prod: azure-dotnet
+  ms.date: 03/04/2020
+
+highlightedContent:
+# itemType: architecture | concept | deploy | download | get-started | how-to-guide | learn | overview | quickstart | reference | tutorial | whats-new 
+  items:
+    - itemType: quickstart
+      title: Deploy an ASP.NET web app
+      url: https://docs.microsoft.com/azure/app-service/app-service-web-get-started-dotnet
+    - itemType: quickstart
+      title: Build a serverless function
+      url: https://docs.microsoft.com/azure/azure-functions/functions-create-your-first-function-visual-studio
+    - itemType: quickstart
+      title: ASP.NET Core and Docker
+      url: https://docs.microsoft.com/visualstudio/containers/container-tools?view=vs-2019
+    - itemType: learn
+      title: Azure fundamentals
+      url: https://docs.microsoft.com/learn/paths/azure-fundamentals/
+    - itemType: tutorial
+      title: Deploy a .NET app with Azure DevOps
+      url: https://docs.microsoft.com/en-us/azure/devops/pipelines/ecosystems/dotnet-core?view=azure-devops
+    - itemType: tutorial
+      title: Full stack authentication end-to-end in App Service
+      url: https://docs.microsoft.com/azure/app-service/app-service-web-tutorial-auth-aad
+    - itemType: learn
+      title: Secure custom APIs with Microsoft Identity
+      url: https://docs.microsoft.com/learn/modules/identity-secure-custom-api/
+    - itemType: download
+      title: Sample code
+      url: https://docs.microsoft.com/samples/browse/?products=azure&languages=csharp%2Caspx-csharp%2Cfsharp%2Cvb
+
+conceptualContent:
+# itemType: architecture | concept | deploy | download | get-started | how-to-guide | learn | overview | quickstart | reference | tutorial | video | whats-new
+  items:
+    - title: Build web apps
+      links:
+        - itemType: learn
+          text: Host a web application with Azure App Service
+          url: https://docs.microsoft.com/learn/modules/host-a-web-app-with-azure-app-service/
+        - itemType: tutorial
+          text: ASP.NET Core web app with App Service and Azure SQL Database
+          url: https://docs.microsoft.com/azure/app-service/app-service-web-tutorial-dotnet-sqldatabase
+        - itemType: tutorial
+          text: Managed identity with ASP.NET and Azure SQL Database
+          url: https://docs.microsoft.com/azure/app-service/app-service-web-tutorial-connect-msi
+        - itemType: tutorial
+          text: Web API with CORS in Azure App Service
+          url: https://docs.microsoft.com/azure/app-service/app-service-web-tutorial-rest-api
+        - itemType: learn
+          text: Publish and manage your APIs with Azure API Management
+          url: https://docs.microsoft.com/learn/modules/publish-manage-apis-with-azure-api-management/
+    - title: Build cloud native apps
```

Comment:
Is "cloud native apps" a well understood term?

Reply:
@spboyer please comment?

Reply:
@KrzysztofCwalina - "Cloud Native" is a well known and established industry term.